CHANGE_LOG

```diff
@@ -1,3 +1,42 @@
+Version 0.23.0
+--------------
+
+This release introduces JIT classes using the new ``@jitclass`` decorator,
+allowing user-defined structures for nopython mode.  Other improvements
+and bug fixes are listed below.
+
+Improvements:
+
+* PR #1571: Implement JIT classes
+* PR #1584: Improve typing of array indexing
+* PR #1583: Allow printing booleans
+* PR #1542: Allow negative values in np.reshape()
+* PR #1560: Support vector and matrix dot product, including ``np.dot()``
+  and the ``@`` operator in Python 3.5
+* PR #1546: Support field lookup on record arrays and scalars (i.e.
+  ``array['field']`` in addition to ``array.field``)
+* PR #1440: Support the HSA wavebarrier() and activelanepermute_wavewidth()
+  intrinsics
+* PR #1540: Support np.angle()
+* PR #1543: Implement parallel gufuncs
```

Comment:
Can we describe this as `Implement CPU multithreaded gufuncs (target="parallel")`